.github/workflows/project_thread_update_reminders.yml

```diff
@@ -13,20 +14,34 @@ on:
         default: true
 
 jobs:
-  check:
+  send-reminders:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
 
-      - name: Set up Node.js
-        uses: actions/setup-node@v2
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
         with:
+          # Ensure that it matches `engines` field in the following files:
+          # - `package.json`
           node-version: "16"
+          cache: pnpm
+          cache-dependency-path: |
+            **/pnpm-lock.yaml
+            **/.npmrc
+
+      # Install dependencies
+      - name: Install JS dependencies
+        shell: bash
+        run: pnpm install
```

Comment:
Can you use `setup-env` for this?

I think all the environment setup be avoidable if we used https://github.com/actions/github-script instead as it already has the two dependencies available via the parameters passed to the exported function.

Reply:
I'll look into [actions/github-script](https://github.com/actions/github-script), that seems excellent, especially with this option:

https://github.com/actions/github-script#run-a-separate-file

Reply:
It works great!